library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
Yes, its good. We can do this. But alternatively , one more thing can be done .Whenever I need to use Bootstrap over a page , I'll need 3 files, Bootstrap.min.css, Boostrap.min.js, jquery. So , one function to include boostrap can be written here to include these files. Then , later when working over specific pages in UI, I'll remove the jquery version used there as it would be already in the function defined in this file. What about this?